### PR TITLE
Update fastx-toolkit for newer intel compilers.

### PR DESCRIPTION
**Pull request by ehiggs**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
looks ok to me

**Comment by JensTimmerman** [comment]
ok

File: easybuild/easyconfigs/l/libgtextutils/libgtextutils-0.6.1-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCE_TAR_BZ2]
+source_urls = ['http://hannonlab.cshl.edu/fastx_toolkit']
+
+sanity_check_paths = {
+                      'files': ['lib/libgtextutils.so', 'lib/libgtextutils.a'],
+                      'dirs': ['lib/pkgconfig']
+                     }
```
Comment (boegel) [inline easybuild/easyconfigs/l/libgtextutils/libgtextutils-0.6.1-intel-2015a.eb]: please fix indent here

File: easybuild/easyconfigs/l/libgtextutils/libgtextutils-0.6.1-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+easyblock = 'ConfigureMake'
+
+name = 'libgtextutils'
+version = '0.6.1'
+
+homepage = 'http://hannonlab.cshl.edu/fastx_toolkit/'
+description = """ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream"""
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/l/libgtextutils/libgtextutils-0.6.1-intel-2015a.eb]: drop an empty line

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/fea75a8fc1d8fa8ae7f9 for a full test report.

**Comment by boegel** [comment]
Thanks @ehiggs!

**Pull request merged**